crates/sui-tool/src/commands.rs

```diff
@@ -396,8 +396,20 @@ pub enum ToolCommand {
         safety_checks: bool,
         #[arg(long = "authority")]
         use_authority: bool,
-        #[arg(long = "cfg-path", short)]
+        #[arg(
+            long = "cfg-path",
+            short,
+            help = "Path to the network config file. This should be specified when rpc_url is not present. \
+            If not specified we will use the default network config file at ~/.sui-replay/network-config.yaml"
+        )]
         cfg_path: Option<PathBuf>,
+        #[arg(
+            long,
+            help = "The name of the chain to replay from, could be one of: mainnet, testnet, devnet.\
+            When rpc_url is not specified, this is used to load the corresponding config from the network config file.\
+            If not specified, mainnet will be used by default"
+        )]
+        chain: Option<String>,
```

Comment:
FWIW, in Sui CLI we refer to this as `env`, e.g., `sui client switch --env devnet`. I like `chain` a bit better than `env`, but don't want to change it in the main CLI yet as it would break a lot of workflows that use env.

Reply:
This is only an option in sui-tool though, not in the main CLI.

Reply:
Yes, but my thought here was that it'd be great to keep them consistent across all CLIs we have.

Reply:
I don't have a strong opinion on this particular option, happy to make the change.
But I do think that the builder/dev facing CLI will always have different options and commands than sui-tool which is an internal tool and should only be used by us or node operators.